MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -5984,16 +5998,21 @@ public void addKill(Kill k) {
     }
 
     public List<Kill> getKills() {
-        return Collections.unmodifiableList(kills);
+        List<Kill> flattenedKills = new ArrayList<Kill>();
+        for(ArrayList<Kill> personKills : kills.values()) {
+            flattenedKills.addAll(personKills);
+        }
+        
+        return Collections.unmodifiableList(flattenedKills);
     }
 
     public ArrayList<Kill> getKillsFor(UUID pid) {
```

Comment:
I know this is existing code, but does `getKillsFor` need to be an ArrayList or could it just be a List like `getKills`?